app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -40,17 +43,29 @@ class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterf
     storyId = intent.getStringExtra(EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY)
     explorationId = intent.getStringExtra(EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY)
     backflowScreen = intent.getIntExtra(EXPLORATION_ACTIVITY_BACKFLOW_SCREEN_KEY, -1)
-    explorationActivityPresenter.handleOnCreate(this, internalProfileId, topicId, storyId, explorationId, backflowScreen)
+    explorationActivityPresenter.handleOnCreate(
+      this,
+      internalProfileId,
+      topicId,
+      storyId,
+      explorationId,
+      backflowScreen
```

Comment:
Rename it by following camel-casing `backFlowScreen`

Reply:
@veena14cs 
This is actually correct. Android studio is showing this incorrect but this is a valid word with the correct [meaning](https://www.google.com/search?q=backflow&oq=backflow&aqs=chrome.0.69i59j0l4j69i61l3.1879j0j7&sourceid=chrome&ie=UTF-8) that we want here.

Reply:
Its similar to `OnBoarding` and `Onboarding` where android studio suggests the first one but second is the correct one.

Reply:
This is not part of this PR and also backflowScreen is correct as it can be used as a single word.

Reply:
Got it.